### PR TITLE
:building_construction: Consultancy Page About Section (first) #179

### LINKED ISSUE #179: 🎨 Create Consultancy Page About Section (first)
#  Create Consultancy Page About Section (first)
---

#### Note: This issue is dependent on #8, please complete that before you continue.

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user, I want a component that displays the first section of the about section on the consultancy page. I want a good design on both desktop and mobile.

### Description

Your assignment is to implement the component to appear as below.  This component reuses the banner component created in #8. Please take note of all the design features.

text image component

#### Link to full design:
```
https://xd.adobe.com/view/b98caf27-6549-49e5-8ed8-c5234788df61-ecdf/
```
### Technical Analysis

This component will be created in `libs\features\pages\consultancy-page` inside a components folder. The component will be called elewa-consultancy-about-one. The component reuses the image and text banner and is therefore dependent on #8. If you need to edit the code in #8 to get the desired result feel free to do so.

The section has a property of the below type:

```typeScript
 ImageAndText
```

NB: The interface for the type above will be implemented in #8

### Data
The mock data for this should be stored in `libs\data\sections`

### Acceptance Criteria

1. When rendered the component is pixel-perfect compared to the design above.
2. The component is clean and does not have more than 50 lines
3. The component SCSS uses CSS variables for colour, font, ... Connected to the main style.scss
6. The design is responsive for screen, tablet and phone
7. The image renders fast and has multiple srcsets

### PR DESCRIPTION
**Pull request by preston-56**
# Description

 - Created `elewa-consultancy-about-one` component.
 - This component reuses `the image and text banner`.

Fixes #179
## Screenshot
Screenshot from 2023-09-08 13-07-39

## Checklist

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      "
+    />
+  </div>
+  <div class="text-container">
+    <!-- Use content -->
+    <elewa-website-elewa-text-content-item
+      [content]="content"
+    ></elewa-website-elewa-text-content-item>
+  </div>
+</div>
```
Comment (AnnaL001) [inline libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.html:19]: As specified, the text and image banner component should be reused

File: libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
+/* Default styles for larger screens */
+.image-text-banner {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 20px;
+  background: #dbc2f5 0% 0% no-repeat padding-box;
```
Comment (AnnaL001) [inline libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.scss:7]: For reuse of colors, use CSS variables specified in elements/theming/_variables.scss when specifying colors/background colors

File: libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.scss [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  border-radius: 50%; /* Create a vertical oval shape */
+}
+
+.text-container {
+  flex: 1;
+  padding: 20px;
+  width: 300px;
+  height: 200px;
+  text-align: left;
+  font: normal normal normal 32px/42px DM Sans;
```
Comment (AnnaL001) [inline libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.scss:30]: Also for fonts, use CSS variables specified in elements/theming/_variables.scss

File: libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.scss [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  letter-spacing: 0px;
+  color: #000000;
```
Comment (AnnaL001) [inline libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.scss:32]: Same here

File: libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.scss [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  opacity: 1;
+  margin-right: 50px;
+  margin-bottom: 350px;
+}
+
+elewa-website-elewa-text-content-item {
+  font-size: 18px;
+  line-height: 1.5;
+  color: #333;
```
Comment (AnnaL001) [inline libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.scss:41]: Same here

**Review by AnnaL001 — CHANGES_REQUESTED** [review]
Good job so far, just a few adjustments

File: libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.scss (hunk 2 above)
Comment (preston-56) [reply libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.scss]: @AnnaL001 *resolved

File: libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.scss (hunk 3 above)
Comment (preston-56) [reply libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.scss]: @AnnaL001 *resolved

File: libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.scss (hunk 4 above)
Comment (preston-56) [reply libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.scss]: @AnnaL001 *resolved

File: libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.scss (hunk 5 above)
Comment (preston-56) [reply libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.scss]: @AnnaL001 *resolved

File: libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.ts [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+  @Input() ImageAndText!: ImageAndText;
+
+  // Use the ContentText type for the content input
+  @Input() content: ContentText = {
+    title: 'People at the center',
+    descriptions: [
+      'People are the secret to successful education programs. Through in-depth research, we work with your organization at different levels to understand (“elewa”) how people live and learn.',
+      'Based on decades of research, this understanding allows us to co-create strong learning networks within your organization. Networks that can be re-used every time a training need arises; At a fraction of the cost of current solutions.',
+    ], // An array of strings
+  };
```
[the remark this answers is not in the document]
Comment (preston-56) [reply libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.ts]: @AnnaL001 *resolved

File: libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.html (hunk 1 above)
Comment (preston-56) [reply libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.html]: @AnnaL001 *resolved

**Review by LemmyMwaura — APPROVED** [review]

**Pull request merged**